examples/DivisorCurve.jl

```diff
@@ -1,5 +1,6 @@
 export AffineCurveDivisor, ProjCurveDivisor, iseffective, multiplicity,
-       divisor, curve, degree
+       divisor, curve, degree, divisor_ideals, global_sections,
+       _global_sections_ideals
```

Comment:
Why export `_global_sections_ideals`, isn't it an internal function? If it is not just internal, why the leading underscore in the name?

Reply:
I forgot to remove it from the export functions, it is supposed to be an internal one, I put it there to debug something and forgot it after...